scripts/checkpoint_converters/convert_mixtral_hf_to_nemo.py

```diff
@@ -342,5 +393,21 @@ def convert(args):
 
 if __name__ == '__main__':
     args = get_args()
+    if args.low_ram:
+        os.makedirs(args.tmp_dir, exist_ok=True)
+
     parallel_state.set_expert_model_parallel_world_size(1)
-    convert(args)
+    checkpoint = OrderedDict()
```

Comment:
Can you add some comments how the below change helps low ram? 
- load partial and save partial => then load and merge? how did it save mem

Reply:
Sure, memory reduction is achieved by not having the full converted checkpoint and the huggingface checkpoint on memory concurrently, instead you save the converted checkpoint layer by layer and then remove HF checkpoint from memory and then final save to .nemo format.